docs/CORE_API.md

```diff
@@ -173,3 +173,18 @@ Returns the element which received the last fired event.
 - `currentEvent()`
 
 Returns the last fired event.
+
+
+## Ajax
```

Comment:
we should mention this also in the README, in the "summary" section (which links to this doc): https://github.com/ralixjs/ralix#core-methods